src/index.js

```diff
@@ -1,17 +1,35 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom/client";
+import { BrowserRouter } from "react-router-dom";
+import App from "./App";
+import { Auth0Provider } from "@auth0/auth0-react";
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+
+const theme = createTheme({
+  palette: {
+    primary: {
+      main: "#4653F6",
+    },
+    secondary: {
+      main: "#298358",
+    },
+  },
+});
+
+const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+  <Auth0Provider
+    domain="dev-1rg0lsyqba64r0xt.us.auth0.com"
+    clientId="EtZRevz9ac0kFWOwKg0USVNEej62z0aL"
+    authorizationParams={{
+      redirect_uri: window.location.origin,
+    }}
+  >
+    <BrowserRouter>
+      <ThemeProvider theme={theme}>
```

Comment:
Same we can encorporate this components in a parent component, of some sort so that it persists in all the pages